### DISCUSSION TITLE
Anyone using Fab-Manager for other types of services, communities or clubs?

### DISCUSSION THREAD

**Discussion by user90210** [sleede/fab-manager#464] [General]
Are you using or are you aware of deployments of fab-manager for organisations other than Fablabs?

How viable would it be for use in a Sports Club setting?

**Comment by microbearlogist** [comment]
We are actually using fab-manager in a laboratory for equipment and room booking (with the option of making access to certain areas conditional on training).
We don't use the billing function at the moment, but may do so in the future.


The only weakness of Fab-manager is the management of reservation calendars on the administrator's side if you want to have daily reservation slots for a year or more.
